src/interface/comm.rs

```diff
@@ -166,7 +166,7 @@ pub fn new_sockaddr_unix(family: u16, path: &[u8]) -> SockaddrUnix {
 }
 
 pub fn gen_ud_path() -> String {
-    let mut owned_path: String = "/tmp/sock".to_owned();
+    let mut owned_path: String = "/sock".to_owned();
```

Comment:
I forget why we changed this to just /sock from /tmp/sock.

It would make sense to go in /tmp, we should check this again to see if that breaks anything.